tools/ci/tc/tasks/test.yml

```diff
@@ -119,6 +123,7 @@ components:
       - python3.9
       - python3.9-distutils
```

Comment:
Are distutils really needed? If it is, shouldn't it be in the setup instructions for users too?